### DISCUSSION TITLE
Customize the position and size of the control window.

### DISCUSSION THREAD

**Discussion by yxm9264** [rxhanson/Rectangle#1154] [General]
Feature ideas are now kept in the [discussions tab](https://github.com/rxhanson/Rectangle/discussions). Please create an issue there, not here.I hope to be able to customize the position and size of the control window (using keyboard shortcuts), as currently all window positions and sizes are fixed and cannot be changed.

Thank you!

**Comment by SavePointSam** [comment]
I would like to be able to set a custom width and height, then have the window set that size and center on screen upon keypress.

I currently use the application for a Macbook that I sometimes plug into an external monitor. While on this external monitor, the default sizes end up being too large and I end up manually resizing.
